Packs/Cyberint/Integrations/Cyberint/Cyberint.py

```diff
@@ -504,20 +540,53 @@ def fetch_incidents(client: Client, last_run: Dict[str, int],
                         attachments.append(tmp_attachment)
 
         alert["attachments"] = attachments
-        alert_csv_id = alert.get('alert_data', {}).get('csv', {}).get('id', '')
-        if alert_csv_id:
-            extracted_csv_data = extract_data_from_csv_stream(client, alert_id,  # type: ignore
-                                                              alert_csv_id)
-            alert['alert_data']['csv'] = extracted_csv_data
+        incident = {}
+
+        incident_csvs = dict_safe_get(alert, ['alert_data', 'csv', 'content']) or []
+
+        if not fetch_assign_multiple:
+            if incident_csvs:
+                index = 1
+                for incident_csv in incident_csvs:

```

Comment:
```suggestion
                for index, incident_csv in enumerate(incident_csvs):
```
Let's use the enumerate() function! 
It allows us to loop over something and have an automatic counter!

Reply:
Great! Thanks.